cardano-node/cardano-node.cabal

```diff
@@ -252,7 +252,7 @@ test-suite cardano-node-test
   build-depends:        base
                       , aeson
                       , bytestring
-                      , cardano-crypto-class
+                      , cardano-crypto-class ^>=2.2.3.2
```

Comment:
This line isn't needed, since `cardano-node-test` is depending on the `cardano-node` library already.